plugins/lint/current_lint.ml

```diff
@@ -0,0 +1,86 @@
+open Current.Syntax
+module Docker = Current_docker.Default
+
+let src = Logs.Src.create "current_fmt" ~doc:"Current pipelines for linting"
+
+module Log = (val Logs.src_log src : Logs.LOG)
+
+let option_map f = function Some x -> Some (f x) | None -> None
+
+let version_from_string =
+  let re =
+    Re.(
+      seq
+        [
+          start;
+          rep space;
+          str "version";
+          rep space;
+          char '=';
+          rep space;
+          group (rep1 @@ diff graph (set "#"));
+          rep space;
+          eol;
+        ]
+      |> compile)
+  in
+  fun path -> Re.exec_opt re path |> option_map (fun g -> Re.Group.get g 1)
+
+let version_from_file path =
+  let ( let+ ) = Lwt.Infix.( >>= ) in
+  let ( let* ) = Lwt.Infix.( >|= ) in
+  if not (Sys.file_exists path) then
+    let () = Logs.info (fun m -> m "No .ocamlformat file found") in
+    Lwt.return (Ok None)
+  else
+    let+ channel = Lwt_io.open_file ~mode:Lwt_io.input path in
+    let* versions =
+      Lwt_io.read_lines channel
+      |> Lwt_stream.filter_map version_from_string
+      |> Lwt_stream.to_list
+    in
+    match versions with
+    | [ v ] ->
+        let () =
+          Logs.info (fun m -> m "Found OCamlformat version '%s' in dotfile" v)
+        in
+        Ok (Some v)
+    | _ -> Error (`Msg "Unable to parse .ocamlformat file")
+
+let get_ocamlformat_version ~src =
+  Current.component "Infer OCamlformat\nversion"
```

Comment:
You should probably use `@,` to force a line-break here. Though it might be better to use `@ ` as a break hint and let it wrap it itself. If it's not wrapping in a sensible place, it probably just needs a change to `component` to set a sensible default wrap width.